page/partials/faq.ejs

```diff
@@ -1,6 +1,36 @@
-<div class="section whitepaper ecosystem">
-    <div class="text-box">
-        <h5>Coming soon</h5>
-        <div class="underline"></div>
-    </div> <!-- end text box -->
-</div>
+<div class="faq">
+    <div class="row">
+        <!--Nav Bar -->
+        <nav class="col-lg-4 col-md-5 bs-docs-sidebar hidden-sm hidden-xs">
+            <ul id="sidebar" class="sidebar nav nav-stacked fixed">
+            <% for ( var i = 0 ; i < faq.length ; i++ ) {
+                var question = faq[i];
+                %>
+                <li><a class="smooth-scroll" href="#<%=question.title.replace(/[^a-zA-Z ]/g, '').replace(/\s+/g, '-').toLowerCase();%>"><%=question.title%></a></li>
```

Comment:
I would create function that generate this href as separate function then template here would be more readable.